src/Raven.Studio/typescript/common/generalUtils.ts

```diff
@@ -406,6 +406,16 @@ class genUtils {
         return inputNumber.toLocaleString(undefined, { minimumFractionDigits: n, maximumFractionDigits: n });
     };
 
+    /***  File Methods ***/
+    
+    static isFileOfType(fileName: string, possibleFileTypes: string[]): boolean {
```

Comment:
+ I would be nice to have word extension in method name. 

Should we are checking if given path points to file with given extension.

Reply:
done - using getFileExtension method